collections/inherent-io/keycloak.md

```diff
@@ -0,0 +1 @@
+Keycloak support : parser and brute-force detection
```

Comment:
Provide a configuration example? Does it log to a file if so where is the file location?

```
filenames:
  - /path/to/log
labels:
  type: keycloak
  ```
  Is it a docker program
 ```
 source: docker
 container_name:
   - keycloak
 labels:
   type: keycloak
   ```